Sources/AppDelegate.swift

```diff
@@ -33,7 +36,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         enableBackgroundFetch()
+        mainViewController = container.resolve(MainViewController.self)
         
+        let window = UIWindow(frame: UIScreen.mainScreen().bounds)
```

Comment:
window is optioneel (per ApplicationDelegate protocol) ik vind dit zelf mooier dan bij elke setter op window een `?` erachter. `window?.backgroundColor = UIColor.whiteColor()`